frontend/src/Components/ClassLayout.tsx

```diff
@@ -21,6 +22,7 @@ import StatsCard from './StatsCard';
 
 function ClassInfoCard({ classInfo }: { classInfo?: Class }) {
     const navigate = useNavigate();
+    const thisSelectedClasStatus = classInfo?.status;
```

Comment:
@corypride Is this variable necessary or can `classInfo?.status` be used wherever it's accessed?